hive/server/condenser_api/objects.py

```diff
@@ -120,6 +120,10 @@ def _condenser_post_object(row, truncate_body=0):
     """Given a hive_posts_cache row, create a legacy-style post object."""
     paid = row['is_paidout']
 
+    # condenser#3424 mitigation
+    if not row['category']:
+        row['category'] = 'undefined'
```

Comment:
Why not use `None` here? Is it important for it to be a string? If so, why not a blank string? 'undefined' is a weird thing to set it to and could be confusing.

Reply:
`condenser#3424` shows issues with using a blank value, so this sets what the UI should display. `None` or a blank string would result in the same condition. Right now this would result in condenser reading `Posted by @username in undefined 2 hours ago` (as opposed to `Posted by @username in 2 hours ago`). I considered using some other generic label, but IMO it's a bad practice for UIs to leave this field blank so I left it something obvious.